ssh.go

```diff
@@ -73,6 +73,11 @@ func (c *SSH) IPAddress() string {
 // String returns the connection's printable name
 func (c *SSH) String() string {
 	if c.name == "" {
+		// Pritify IPv6 print
+		if strings.Contains(c.Address, ":") {
```

Comment:
Maybe make something like `c.isIPv6()` to avoid repeating the `strings.Contains`

Reply:
Yes agree on this. Any advice of where to make the initial Check ? to set isIPv6